### ISSUE TITLE
Expansion proof viewer: spooky action at a distance

### ISSUE THREAD

**Issue by gebner** [gapt/gapt#550]
In this example with two quantifiers, clicking on the weak quantifier automatically expands the strong one as well and vice versa:
```scala
prooftool(Escargot getExpansionProof hof"∃x p x | ¬∃x p x")
```

**Comment by gebner** [comment]
This issue also arises when expanding cuts:
```scala
prooftool(LKToExpansionProof(CutIntroduction.compressLKProof(LinearExampleProof(9)).get))
```

**Comment by loewenheim** [comment]
Fixed in 7800fe28def061a470b71e476cac586600be028c.
